parachain/pallets/ethereum-client/fixtures/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "snowbridge-pallet-ethereum-client-fixtures"
+description = "Snowbridge Ethereum Client Test Fixtures"
+version = "0.9.0"
+authors = ["Snowfork <contact@snowfork.com>"]
+edition.workspace = true
+repository.workspace = true
+license = "Apache-2.0"
+categories = ["cryptography::cryptocurrencies"]
+
+[lints]
+workspace = true
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+hex-literal = { version = "0.4.1" }
+sp-core = { path = "../../../../polkadot-sdk/substrate/primitives/core", default-features = false }
+sp-std = { path = "../../../../polkadot-sdk/substrate/primitives/std", default-features = false }
+frame-benchmarking = { path = "../../../../polkadot-sdk/substrate/frame/benchmarking", default-features = false, optional = true }
+frame-support = { path = "../../../../polkadot-sdk/substrate/frame/support", default-features = false }
+frame-system = { path = "../../../../polkadot-sdk/substrate/frame/system", default-features = false }
+snowbridge-core = { path = "../../../primitives/core", default-features = false }
+snowbridge-beacon-primitives = { path = "../../../primitives/beacon", default-features = false }
+
+[features]
+default = ["std"]
+std = [
+	"frame-benchmarking/std",
+	"frame-support/std",
+	"frame-system/std",
+	"snowbridge-beacon-primitives/std",
+	"snowbridge-core/std",
+	"sp-core/std",
+	"sp-std/std",
+]
+runtime-benchmarks = [
+	"frame-benchmarking",
+	"frame-benchmarking/runtime-benchmarks",
+	"frame-support/runtime-benchmarks",
+	"frame-system/runtime-benchmarks",
+	"snowbridge-core/runtime-benchmarks",
+]
```

Comment:
`runtime-benchmarks` seems not necessary here.

Reply:
I referenced the inbound fixtures crate you added. 😅 https://github.com/Snowfork/snowbridge/blob/main/parachain/pallets/inbound-queue/fixtures/Cargo.toml#L38 Is it necessary in that crate?

Reply:
Yeah, seems also unnecessary in that crate because IIUC we're using the fixture from local setup for the benchmark?

The benefit of introducing the `runtime-benchmarks` feature is to differentiate the fixture between public network(goerli/sepolia) and local setup(hacked mainnet), As we know the fork version is different and we may want to use the real data(from public network) for the benchmark.

So it's nitpicky and feel free to ignore it.